docs/introduction.rst

```diff
@@ -65,23 +67,32 @@ in which many of the nodes and individuals have been removed.
 Who and what is in the tree sequence?
 *************************************
 
-OK, who and what exactly *is* left in the tree sequence?
+OK, who and what exactly *is* left in the tree sequence after simplification?
 We clearly want all individuals of the *final* generation
 to have their genealogies recorded in the tree sequence.
-in the picture above there is one additional individuals pictured
-in the tree sequence that was not alive at the end of the simulation,
-because it was *remembered*, using the ``treeSeqRememberIndividuals()`` method.
+in the picture above there are two additional individuals pictured in the
+tree sequence that were not alive at the end of the simulation, because they
+were *remembered*, using the ``treeSeqRememberIndividuals()`` method. The lower
+one (still containing 2 nodes or chromosome) was *permanently* remembered, which is the
+default behaviour. The upper one (with only one node left) was simply marked as *retained*,
+done by passing ``permanent=F`` when calling ``treeSeqRememberIndividuals()``.
+As you might expect, permanently remembered individuals and their nodes are always
+kept in the tree sequence after simplification. In contrast, individuals that are
+merely marked for retention can lose their nodes (as has happened to one of the nodes in
+this case). Furthermore, once all an individuals's nodes are lost, simplification
+removes the individual too, for efficiency reasons.
```

Comment:
Suggested edit:
"
In the picture above there are two additional individuals pictured in the
tree sequence that were not alive at the end of the simulation. These are still present because they
were *remembered*, using the ``treeSeqRememberIndividuals()`` method. The lower
one (still containing 2 nodes or chromosome) was *permanently* remembered, which is the
default behaviour. The upper one (with only one node left) was only "retained",
by passing ``permanent=F`` when calling ``treeSeqRememberIndividuals()``.
As you might expect, permanently remembered individuals and the entire genealogical history of both their nodes are always
kept in the tree sequence, while individuals that are
merely retained will only be present in the tree sequence as long as at least one of their nodes
has not been removed by simplification.
"

Hm. I'm kinda feeling like this is the wrong place for a technical discussion, being the *introduction*, and maybe "retaining" is a more advanced topic to be explained elsewhere. What do you think? The problem is that the above paragraph still isn't quite there, because the fundamental difference between permanent/not is that 
- a permanently remembered individual is guaranteed to always have the entire history of both their chromosomes in the tree sequence
- whereas a retained individual only acts as some extra annotation for its nodes: the only parts of its genomes that are actually recorded in the tree sequence are those on which it's the MRCA of two samples, unless `keep_unary` is on.

How about we remove mention of "retaining" from the Introduction, and make a new section in the Tutorial about it?

Reply:
I think it's useful to have it for the picture (because we need a picture for showing the difference between retained and remembered).

Also, if we are removing mention of retaining, should we not also remove a mention of Remembering entirely from the intro, as many people will use pyslim without Remembering or Retaining?

I wonder if we should make Remembering (+ retaining) a separate section on their own, perhaps in the intro?